### DISCUSSION TITLE
Option to Delete Media associated with the Posts when we delete the Post/Fact Check

### DISCUSSION THREAD

**Discussion by atomsalt** [factly/dega#586] [Ideas]
General - Option to Delete Media associated with the Posts when we delete the Post/Fact Check

**Comment by deshetti** [comment]
The same media file could be used across multiple posts. So, if we were to delete the media - we should also confirm that it is not used anywhere else prior to deleting it. 

We need to wait on implementing this feature based on if there is a need from users IMO. Converting this into a discussion for right now.

We can develop a cleanup service that permanently deletes all the content once it is over 30 days in the trash. This should include dangling content in S3 like Media. To check for Media that is not used anywhere requires searching in all the entities where the Media could be used and it is an expensive query as the database and media list grows.
